packages/nimble-components/src/rich-text-editor/tests/rich-text-editor-labels.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { html } from '@microsoft/fast-element';
+import { richTextEditorTag, type RichTextEditor } from '..';
+import { type Fixture, fixture } from '../../utilities/tests/fixture';
+import { themeProviderTag, type ThemeProvider } from '../../theme-provider';
+import {
+    LabelProviderRichTextEditor,
+    labelProviderRichTextEditorTag
+} from '../../label-provider/rich-text-editor';
+import { RichTextEditorPageObject } from '../testing/rich-text-editor.pageobject';
+import { LabelProvider, ToolbarButton } from '../testing/types';
+import { getSpecTypeByNamedList } from '../../utilities/tests/parameterized';
+import { waitForUpdatesAsync } from '../../testing/async-helpers';
+
+async function setup(): Promise<Fixture<ThemeProvider>> {
+    return fixture<ThemeProvider>(
+        html`
+      <${themeProviderTag}>
+          <${labelProviderRichTextEditorTag}></${labelProviderRichTextEditorTag}>
+          <${richTextEditorTag}></${richTextEditorTag}>
+      <${themeProviderTag}>`
+    );
+}
+
+const formattingButtons: {
+    name: string,
+    property: LabelProvider,
+    label: string,
+    toolbarButtonIndex: ToolbarButton
```

Comment:
Avoid referring to this as index

```suggestion
    toolbarButton: ToolbarButton
```